### ISSUE TITLE
CLI version command does not give actionable instruction in case of potentially breaking update.

### ISSUE THREAD

**Issue by bender2k14** [CONNECT-platform/codedoc#20]
Executing `codedoc version` says to execute `codedoc update`, which seems to execute successfully, but executing `codedoc version` again still says to execute `codedoc update`.

```
twilliams@EDMONDS MINGW64 /c/Code/Blogs/tyson-williams-blog (master)
$ codedoc version

# Fetching version info ...
# @codedoc/cli version 0.2.0 ...................................... latest: 0.2.0
# @codedoc/core version: 0.1.63 ................................... latest: 0.2.0

# WARNING: some packages are missing or need update.
# Please run codedoc update


twilliams@EDMONDS MINGW64 /c/Code/Blogs/tyson-williams-blog (master)
$ codedoc update

# Updating CLI ...
# Updating local codedoc installation ...


twilliams@EDMONDS MINGW64 /c/Code/Blogs/tyson-williams-blog (master)
$ codedoc version

# Fetching version info ...
# @codedoc/cli version 0.2.0 ...................................... latest: 0.2.0
# @codedoc/core version: 0.1.63 ................................... latest: 0.2.0

# WARNING: some packages are missing or need update.
# Please run codedoc update
```

**Comment by loreanvictor** [comment]
this is because `codedoc update` cannot/should not update to a version that is potentially not backwards-compatible (identified by a change in the left-most non-zero number of the version).

you can force update when you are sure that there are no breaking changes (in this case there aren't) using `codedoc update latest` (which is available after `0.2.0`, so people with cli below `0.2.0` should update using `npm i -g @codedoc/cli` to be able to use `update latest`).

I have [updated the CLI docs](https://codedoc.cc/docs/cli#updates) in response to these changes, but will keep this issue open to improve what the CLI instructs. the CLI can actually check if there is a potentially breaking update and instruct accordingly, plus it would help to link to the docs as well.

**Comment by loreanvictor** [comment]
The CLI now checks if there is a major update, and will print-out relevant hints accordingly. This update is deployed via `@codedoc/cli@0.2.1`.
